apps/explorer/components/Transaction/TransactionHeader.tsx

```diff
@@ -27,14 +32,16 @@ export function TransactionHeader({ title, transactionHeaderData }: Props) {
       />
       <div className="flex gap-4 items-center overflow-hidden">
         {!!timestamp && (
-          <Tooltip content={timestamp}>
-            <Text font="mono" color="subtle" ellipsis>
-              {humanDate(timestamp, {
-                dateStyle: 'medium',
-                timeStyle: 'short',
-              })}
-            </Text>
-          </Tooltip>
+          <ClientSideOnly fallback={<div className="animate-pulse">...</div>}>
```

Comment:
I do prefer seeing a good amount of duplication before abstracting, but might be enough instances here for a <ClientDate /> to make sense.